src/components/atoms/TextInput.vue

```diff
@@ -1,38 +1,48 @@
 <script setup>
-import { ref } from 'vue'
-import store from '@/store'
+import { ref } from "vue";
+import store from "@/store";
 
-const inputText = ref('')
-const isInvalidInput = ref(false)
+const inputText = ref("");
+const isInvalidInput = ref(false);
 
-function submitTodo (inputText) {
-  this.isInvalidInput = validateInput(this.inputText)
+function submitTodo(inputText) {
+  this.isInvalidInput = validateInput(this.inputText);
   if (!this.isInvalidInput) {
-    store.commit('setTodo', inputText)
-    this.inputText = ''
+    store.commit("addTodo", inputText);
+    this.inputText = "";
   }
 }
 
-function validateInput (inputText) {
-  return (!inputText || /^\s*$/.test(inputText) || inputText.length > 150)
+function validateInput(inputText) {
+  return !inputText || /^\s*$/.test(inputText) || inputText.length > 150;
 }
 </script>
 
 <template>
-<form @submit.prevent="submitTodo(inputText)">
-    <input :class="{ 'input-invalid input-base' : isInvalidInput === true, 'input-base' : isInvalidInput === false}" v-model.trim="inputText" placeholder="Add task..."/>
-</form>
+  <form @submit.prevent="submitTodo(inputText)">
+    <input
+      class="input-base"
+      :class="{ 'input-invalid': isInvalidInput === true }"
```

Comment:
instead of using condition, you can just use 
{ 'input-invalid': isInvalidInput }